src/services/network-modification-types.ts

```diff
@@ -37,3 +48,307 @@ export interface VscModificationInfo {
+    permanentLimit: number;
+    temporaryLimits: TemporaryLimit[];
+}
+
+export interface SubstationModificationInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    modificationUuid?: UUID;
+    id: string;
+    name: string | null;
+    country: string;
+    properties?: Property[];
+}
+
+export interface VoltageLeveInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    voltageLevelId: string;
+    voltageLevelName: string | null;
+    substationId?: string | null;
+    nominalV: number | null;
+    lowVoltageLimit: number | null;
+    highVoltageLimit: number | null;
+    busbarCount?: number;
+    sectionCount?: number;
+    switchKinds?: any[];
+    couplingDevices?: any[];
+    isUpdate?: boolean;
+    modificationUuid?: UUID;
+    properties?: Property[];
+}
+
+export interface VoltageLeveCreationlInfo extends VoltageLeveInfo {
+    ipMin: number | null;
+    ipMax: number | null;
+}
+export interface VoltageLeveModificationInfo extends VoltageLeveInfo {
+    lowShortCircuitCurrentLimit: number | null;
+    highShortCircuitCurrentLimit: number | null;
+}
+
+export interface AttachmentLine {
+    type: string;
+    equipmentId: string;
+    equipmentName: string | null;
+    r: number;
+    x: number;
+    g1: number;
+    b1: number;
+    g2: number;
+    b2: number;
+    currentLimits1: CurrentLimits;
+    currentLimits2: CurrentLimits;
+}
+type VariationFilter = {
+    id: string;
+    name: string;
+    specificMetadata: { type: string };
+};
+export interface Variations {
+    variationMode: string | null;
+    variationValue: number | null;
+    reactiveVariationMode: string | null;
+    filters: VariationFilter[];
+}
+
+export interface VSCCreationConverterStation {
+    type: MODIFICATION_TYPE;
+    equipmentId: string;
+    equipmentName: string | null;
+    lossFactor: number;
+    voltageSetpoint: number | null;
+    reactivePowerSetpoint: number | null;
+    voltageLevelId: string;
+    busOrBusbarSectionId: string;
+    connectionName: string | null;
+    connectionDirection: string | null;
+    connectionPosition: string | null;
+    voltageRegulationOn: boolean;
+    reactiveCapabilityCurve: boolean;
+    minQ: number | null;
+    maxQ: number | null;
+    reactiveCapabilityCurvePoints: ReactiveCapabilityCurvePointsData[];
+}
+
+export interface VSCModificationConverterStation {
+    voltageSetpoint: AttributeModification<number> | null;
+    lossFactor: AttributeModification<number> | null;
+    reactiveCapabilityCurve: AttributeModification<boolean> | null;
+    busOrBusbarSectionId: AttributeModification<string> | null;
+    type: MODIFICATION_TYPE;
+    minQ: AttributeModification<number> | null;
+    equipmentId: string;
+    reactiveCapabilityCurvePoints:
+        | {
+              p: number | null;
+              oldP: number | null;
+              minQ: number | null;
+              oldMinQ: number | null;
+              maxQ: number | null;
+              oldMaxQ: number | null;
+          }[]
```

Comment:
create a a type in converter-station-utils.tsx ?

Reply:
Done.